distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -171,6 +171,8 @@ type Vector a
        - function: a function taking a `Builder` and adding elements to it.
        - initial_capacity: Initial capacity of the `Builder`; this is for
          pre-allocation and does not affect the size of the resulting `Vector`.
+       - no_warning_propagation: If True, do not gather warnings from values
+         added to the builder.
```

Comment:
Wouldn't it be easier if the paramater was `warning_propagation_enabled` or `propagate_warnings`? Negation in boolean flags usually makes it more complicated as it requires a double negation - if `no_X` is `False` then `X` is enabled. It is much clearer if it is just `X` is true or false.

Reply:
Changed to `propagate_warnings`.